### PR TITLE
[ALLUXIO-3143] Amend getGroupName in AlluxioFuseUtils

### PR DESCRIPTION
**Pull request by murong00**
[https://alluxio.atlassian.net/browse/ALLUXIO-3143](https://alluxio.atlassian.net/browse/ALLUXIO-3143)

Command 'id -ng' displays group name need supplying user's name rather than user id, and there is little point in supplying user's name to command 'id -nu'. Simply use the 'getent' command to get username from uid instead.

### CODE REVIEW

**Comment by apc999** [comment]
Format issue: AlluxioFuseUtils.java:59: error: Line is longer than 100 characters (found 106)

File: integration/fuse/src/main/java/alluxio/fuse/AlluxioFuseUtils.java [hunk 1]
```diff
@@ -56,7 +56,8 @@ public static long getGid(String userName) {
    * @return user name
    */
   public static String getUserName(long uid) throws IOException {
-    return ShellUtils.execCommand("id", "-nu", Long.toString(uid)).trim();
+    // Simply use the 'getent' command to get username from uid and cut the first filed delimited by colon
+    return ShellUtils.execCommand("getent", "passwd", Long.toString(uid)).trim().split(":")[0];
```
Comment (apc999) [inline integration/fuse/src/main/java/alluxio/fuse/AlluxioFuseUtils.java]: different from id, getent may not be available on different platforms, e.g. it is unavailable on MacOS. Can we make it platform dependent?

Comment (yupeng9) [reply integration/fuse/src/main/java/alluxio/fuse/AlluxioFuseUtils.java]: yup, since 1.6 we added support of MacOS to FUSE.

Comment (murong00) [reply integration/fuse/src/main/java/alluxio/fuse/AlluxioFuseUtils.java]: thanks for correcting, command 'id' works for MacOS and Linux 7 while it doesn't support for Linux 6. Linux OS dependency was added for command 'getent', any advise/guidance would be greatly appreciated!

**Comment by LuQQiu** [comment]
Thanks for the improvement!
I didn't notice this PR before until I created and merged the `Fuse user group translation PR` https://github.com/Alluxio/alluxio/pull/7986.

Thanks again for your work!

**Pull request closed without merging**